src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ConflictResolution.targets

```diff
@@ -19,23 +19,18 @@ Copyright (c) .NET Foundation. All rights reserved.
   <Import Project="Microsoft.NET.DefaultPackageConflictOverrides.targets" />
 
   <UsingTask TaskName="ResolvePackageFileConflicts" AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
-  <Target Name="_HandlePackageFileConflicts" DependsOnTargets="GetReferenceAssemblyPaths">
+  <Target Name="_HandlePackageFileConflictsForBuild" DependsOnTargets="GetReferenceAssemblyPaths">
 
-    <ItemGroup Condition="'$(CopyLocalLockFileAssemblies)' != 'true'">
-      <!-- We need to find all the files that will be loaded from deps for conflict resolution.
-          To do this, we look at the files that would be copied local when CopyLocalLockFileAssemblies is true.
-          However, if CopyLocalLockFileAssemblies is true, then we don't add these items, as they
-          will always be included in ReferenceCopyLocalPaths.
-          -->
-      <_LockFileAssemblies Include="@(RuntimeCopyLocalItems)" />
-      <_LockFileAssemblies Include="@(NativeCopyLocalItems)" />
-      <_LockFileAssemblies Include="@(ResourceCopyLocalItems)" />
-    </ItemGroup>
-
-    <!-- Also include RuntimeTarget items, which aren't ever included in ReferenceCopyLocalPaths, but need to be considered
-         for conflict resolution. -->
     <ItemGroup>
-      <_LockFileAssemblies Include="@(ResolvedRuntimeTargets)" />
+      <!--
+        All lock file assemblies for conflict resolution.
+        Exclude the copy-local items since those are passed in separately.
+      -->
+      <_LockFileAssemblies Include="@(RuntimeCopyLocalItems);
```

Comment:
As long as we're refactoring this, I'd rename `_LockFileAssemblies`.  The assets file used to be called the lock file, which is where that name comes from.  Perhaps something like `RuntimeAssetsForConflictResolution` would be good.

Reply:
I'll change.